src/main/java/com/revature/byteshare/Tags/TagController.java

```diff
@@ -10,30 +10,49 @@
 @RestController
 @RequestMapping("/tags")
 public class TagController {
+
     private final TagService tagService;
+    //TODO will need a recipe service reference here to facilitate find recipes by tag
 
     @Autowired
     public TagController(TagRepository tagRepository){
         this.tagService = new TagService(tagRepository);
     }
 
-    @GetMapping
-    public List<Tag> getAllTags(){
-        return tagService.findAllTags();
+    //TODO This is the method that will take a List of Ints from tagservice
+    // and use it to make a list of recipes from recipe service. Will implement ASAP
+    /*
+    @GetMapping("/{tag_name}") // possibly use query param? not entirely sure when to use one over the other tbh
+    public List<Recipe> getAllRecipesByTagName(@PathVariable String tag_name){
+        return
     }
-    @GetMapping("/{recipeId")
+     */
+
+    // TODO : FOR REVIEWER: do we actually just want this to return a list
+    //  of Tag_names or do we want it returning the whole tag object?
```

Comment:
Just the tag name will do

Reply:
Sounds good, I changed that methods to only return the tag names.